api/lib/database/helpers.js

```diff
@@ -94,6 +94,45 @@ const insert = ({ hasura_url, hasura_admin_key }, upsert = false) => (
   return graphQLQuery({ hasura_url, hasura_admin_key })(query)
 }
 
+const insertWithoutPrefix = (
```

Comment:
what is this?

Reply:
Yes, this is repeating too much code. I will instead add a boolean as a parameter for insert, so we can use this function without the `schema _prefix` thing

Reply:
I don't get the problem, can you explain?